src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +85,13 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(i -> i.getSex().equals(Person.Sex.WOMAN)
+                        && i.getAge() >= femaleAge
+                        && i.getCats().size() != 0)
```

Comment:
```suggestion
                 )
```
flatMap() має захендлити пусті лісти і просто з них нічого не поверне